src/vm/costs/cost_functions.rs

```diff
@@ -120,6 +120,7 @@ define_named_enum!(ClarityCostFunction {
     BlockInfo("cost_block_info"),
     StxBalance("cost_stx_balance"),
     StxTransfer("cost_stx_transfer"),
+    StxTransferMemo("cost_stx_transfer"),
```

Comment:
Should this be `unimplemented` for now?

Reply:
this line is gone because I used Unimplemented in src/vm/functions/assets.rs